assets/js/components/Dashboard/PermanentConfiguration/ButtonImage.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ComponentType } from "react";
+import { Button } from "react-bootstrap";
+
+interface ButtonImageProps {
+  fileName: string;
+  label: string;
+  onClick: () => unknown;
+}
+
+const ButtonImage: ComponentType<ButtonImageProps> = (
+  props: ButtonImageProps
+) => {
+  const { fileName, label, onClick } = props;
+  return (
+    <div className="button-image-container">
```

Comment:
The content of this container should be horizontally centered, but isn't

Reply:
Ugh, another case of me only testing with FF. For some reason, Chrome doesn't center them automatically like FF does. Will add explicit centering.